test/Microsoft.AspNetCore.Authentication.Test/CookieTests.cs

```diff
@@ -1247,7 +1247,7 @@ private TestServer CreateServer(Action<CookieAuthenticationOptions> configureOpt
             => CreateServerWithServices(s =>
             {
                 s.AddSingleton<ISystemClock>(_clock);
-                s.AddAuthentication().AddCookie(configureOptions);
+                s.AddAuthentication(o => o.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme).AddCookie(configureOptions);
```

Comment:
Yeah @danroth27 wants it too, I'll add it now since it makes fixing all the tests easier :)